tensorflow/core/lib/random/random_distributions.h

```diff
@@ -95,47 +133,57 @@ template <class Generator>
 class UniformDistribution<Generator, bfloat16> {
  public:
   // The number of elements that will be returned.
-  static constexpr int kResultElementCount = Generator::kResultElementCount;
+  // Set the number to be greater equal to Eigen packet size of type,
+  // so computations can be vectorized using SIMD.
+  static constexpr int kVectorLength =
+      Eigen::internal::packet_traits<bfloat16>::size;
```

Comment:
Wouldn't this get a compilation error since the MR defining `packet_traits<bfloat16>` is not merged into Eigen yet?

Reply:
There's a default template in [Eigen](https://gitlab.com/ShengYang1/eigen/-/blob/bf16_in_eigen/Eigen/src/Core/GenericPacketMath.h#L111), it will be `1` if the given type is  not defined.

Reply:
Thank you!